Plugins/DotNET/DotNETExports.cpp

```diff
@@ -331,6 +346,24 @@ static const char* StackPopString()
     return strdup(String::ToUTF8(value.CStr()).c_str());
 }
 
+static const char* StackPopRawString()
+{
+    auto vm = Globals::VirtualMachine();
+    ASSERT(vm->m_nRecursionLevel >= 0);
+
+    CExoString value;
+    if (!vm->StackPopString(&value))
+    {
+      LOG_WARNING("Failed to pop string - recursion level %i.", vm->m_nRecursionLevel);
+      return nullptr;
```

Comment:
Use `""` instead of `nullptr`. Although I now see above that it might need to be `strdup("")` instead.

Reply:
See my above comment. The intention is that null pointers are handled by the managed code.